### ISSUE TITLE
Integrate profile registry

### ISSUE THREAD

**Issue by a-mile** [ResearchObject/ro-crate#303]
Add the profile registry (under Community?) with a means to submit a profile.

**Comment by elichad** [comment]
@marc-portier @cedricdcc this issue has been moved to the central ro-crate repo now that we've rolled out the new website. It'd be good to renew our conversations about this profile registry integration - at one of the RO-Crate calls or otherwise

**Comment by marc-portier** [comment]
Can you pass the new reference?

**Comment by elichad** [comment]
@marc-portier It's still this thread - the whole thread has been moved using GitHub's 'transfer issue' feature. The URL used to be https://github.com/ResearchObject/ro-crate-2024/issues/13, now it is https://github.com/ResearchObject/ro-crate/issues/303.

**Comment by elichad** [comment]
Hi @marc-portier @cedricdcc are you able to come along to one of the upcoming [RO-Crate meetings](https://calendar.google.com/calendar/u/0/embed?src=b53932af95f34472d7253e3f54e7e11ab1bbe3bf4477c5c60e94db75e08d151e@group.calendar.google.com&ctz=Europe/London) to discuss how to approach this integration? Either the normal community call or the Europe regional drop-in (which I usually host) would work, let me know what's best for you.

**Comment by marc-portier** [comment]
Yes should be doable. Feel free to remind us on the upcoming ones.

**Comment by mpo-vliz** [comment]
@elichad -- could you reach out with some dates that could work for a direct specific call to cover this collaboration? 
from there each of us could report back to the ro-crate meetings with progress or possible issues to resolve

starting point would be to convert the profiles on https://www.researchobject.org/ro-crate/profiles.html into a csv listing residing in a new profile-github-repo (in fact @stain might already have one around?)

**Comment by elichad** [comment]
@mpo-vliz you can view my availability through Calendly here: https://calendly.com/eli-chadwick-uom/55-minute-meeting. Feel free to book a meeting directly through that link, or follow up here with times that work for you (the link books a 55-min meeting but I am happy to keep it shorter than that).

**Comment by stain** [comment]
@OliverWoolland investigating this, adding lists from https://oliverwoolland.github.io/ro-crate-profile-repository/

@elichad to add to this issue additional profiles

**Comment by elichad** [comment]
Additional profiles (not an exhaustive list but the ones I have on hand):

- [BCO RO-Crate](https://biocompute-objects.github.io/bco-ro-crate/recommendation.html)
- [Five Safes Crate](https://trefx.uk/5s-crate/)
- [RO-Crate Convention to Include Schema and Metadata](https://sissource.ethz.ch/sispub/ro-crate/-/blob/main/interoperability/0.1.x/spec.md?ref_type=heads)

Edited 2026-08-25 to add:
- [ISA profile](https://nfdi4plants.github.io/arc-ro-crate-profile/profiles/isa_ro_crate/)
- [ISA-Tox profile](https://github.com/johannehouweling/vitro-crate/blob/main/profiles/docs/isa_tox.md) (associated tooling: [GitHub - johannehouweling/vitro-crate](https://github.com/johannehouweling/vitro-crate))
- [ARC RO-Crate profile collection](https://nfdi4plants.github.io/arc-ro-crate-profile/profiles/)
- other profiles listed on https://www.researchobject.org/ro-crate/profiles.html
